twake/backend/node/src/utils/workspace.ts

```diff
@@ -13,6 +13,14 @@ export async function isWorkspaceAdmin(
     return false;
   }
 
+  const companyUser = await userService.companies.getCompanyUser(
+    { id: workspace.company_id },
+    { id: user.id },
+  );
+  if (companyUser && companyUser.role == "admin") {
```

Comment:
owner also have this right

Reply:
Now you have utils functions to get if this is a "at least" admin level. Search for `hasCompanyAdminLevel` in project